lib/os/cbprintf_packaged.c

```diff
@@ -120,7 +120,7 @@ static int cbprintf_via_va_list(cbprintf_cb out, void *ctx,
 	u.__ap.overflow_arg_area = buf;
 	u.__ap.reg_save_area = NULL;
 	u.__ap.gp_offset = (6 * 8);
-	u.__ap.fp_offset = (6 * 8 + 16 * 16);
+	u.__ap.fp_offset = ((6 * 8) + (16 * 16));
```

Comment:
is this necessary? No, according to: https://wiki.sei.cmu.edu/confluence/display/c/EXP00-C.+Use+parentheses+for+precedence+of+operation
and there is no similar example in https://gitlab.com/MISRA/MISRA-C/MISRA-C-2012/Example-Suite/-/blob/master/R_12_01.c